script/color-contrast.ts

```diff
@@ -30,56 +30,62 @@ const getOpaqueColor = (color: string, background: string): string => {
   // parse BG
   const [bgR, bgG, bgB, bgAlpha] = parseToRgba(background)
   // return mixed color
-  const mixed = normal({ r: bgR, g: bgG, b: bgB, a: bgAlpha }, { r: colorR, g: colorG, b: colorB, a: colorAlpha })
+  const mixed = normal({r: bgR, g: bgG, b: bgB, a: bgAlpha}, {r: colorR, g: colorG, b: colorB, a: colorAlpha})
   return rgba(mixed.r, mixed.g, mixed.b, mixed.a)
 }
 /**
  * runContrastTest
  * @description runs through all color pairs of a theme and checks the contrasts
- * @param colorPairs 
- * @param colors 
+ * @param colorPairs
+ * @param colors
  * @returns contrastTestResult
  */
-const runContrastTest = (colorPairs: ContrastRequirement[], colors: any): contrastTestResult[] =>
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const runContrastTest = (colorPairs: ContrastRequirement[], scopedColors: any): contrastTestResult[] =>
```

Comment:
Why did you rename it to `scopedColors` colors? I agree that `colors` is not the best and maybe `scopedColors` is great, I just don't really understand the name. Can you please explain it to me? 😬

Reply:
thanks for reminding me, I forgot to annotate this one. 

colors is already mentioned in the upper scope, where you're importing the tokens from dist. linters picked this up, so I've renamed this scopedColors to imply it's locally scoped to the function closure.